ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/tasks/PersistBlockTask.java

```diff
@@ -67,22 +68,20 @@ private PersistBlockTask(
       final List<Block> blocks,
       final HeaderValidationMode headerValidationMode,
       final MetricsSystem metricsSystem) {
-    checkArgument(blocks.size() > 0);
+    checkArgument(!blocks.isEmpty());
```

Comment:
nit; adding an error message to the `checkArgument` would make an quicker to understand error condition